pages/c/_cluster/apps/charts/index.vue

```diff
@@ -256,16 +257,14 @@ export default {
     selectChart(chart) {
       let version;
       const OSs = this.currentCluster.workerOSs;
-
       const showPrerelease = this.$store.getters['prefs/get'](SHOW_PRE_RELEASE);
       const compatibleVersions = compatibleVersionsFor(chart, OSs, showPrerelease);
+      const versions = chart.versions;
 
       if (compatibleVersions.length > 0) {
         version = compatibleVersions[0].version;
       } else {
-        console.warn(`Cannot select chart '${ chart.chartName }' (no compatible version available for '${ OSs.length > 1 ? `${ OSs[0] } and ${ OSs[1] }` : OSs[0] }')`); // eslint-disable-line no-console
-
-        return;
+        version = versions[0].version;
```

Comment:
Is it ever possible for versions to be null or undefined? My concern is that we can throw an unhandled error when trying to access `version`.

Reply:
That's a reasonable concern, but chart versions are defined with empty array [here](https://github.com/rancher/dashboard/blob/master/store/catalog.js#L512) then filled so this should be okay.

Reply:
Sounds good, thanks for the head up 😁